src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,33 +2,40 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
 
-    public int total = -1;
+    private static final int EMPTY_STACK_HEAD_INDEX = -1;
+    private static final int CAPACITY = 12;
+    private static final int FULL = 11;
+    private int[] numbers = new int[CAPACITY];
 
-    public void countIn(int in) {
+    private  int total = EMPTY_STACK_HEAD_INDEX;
+
+    protected void countIn(final int in) {
         if (!isFull())
             numbers[++total] = in;
     }
 
-    public boolean callCheck() {
-        return total == -1;
+    protected boolean callCheck() {
+        return total == EMPTY_STACK_HEAD_INDEX;
     }
 
-    public boolean isFull() {
-        return total == 11;
+    protected boolean isFull() {
+        return total == FULL;
     }
 
     protected int peekaboo() {
         if (callCheck())
-            return -1;
+            return EMPTY_STACK_HEAD_INDEX;
```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)